repair/repair.cc

```diff
@@ -756,9 +756,14 @@ future<> repair_task_impl::do_repair_ranges(lw_shared_ptr<repair_info> ri) {
         // repair all the ranges in limited parallelism
         rlogger.info("repair[{}]: Started to repair {} out of {} tables in keyspace={}, table={}, table_id={}, repair_reason={}",
                 ri->id.uuid(), idx + 1, ri->table_ids.size(), ri->keyspace, table_name, table_id, ri->reason);
-        co_await coroutine::parallel_for_each(ri->ranges, [ri, table_id] (auto&& range) {
-            return with_semaphore(ri->rs.get_repair_module().range_parallelism_semaphore(), 1, [ri, &range, table_id] {
-                return ri->repair_range(range, table_id).then([ri] {
+        co_await coroutine::parallel_for_each(ri->ranges, [ri, table_id, parent_data] (auto&& range) -> future<> {
+            auto tmp_ri = ri;
+            auto task_impl_ptr = std::make_unique<range_repair_task_impl>(ri->rs.get_repair_module().shared_from_this(), tasks::task_id::create_random_id(), 0, ri->keyspace, "", format("{}", ri->reason), "", parent_data.id, ri, range, table_id);
```

Comment:
Why does the table here is pass as empty string?

What is the the empty  "" after reason?

Reply:
why is the seq number always zero?

Reply:
Sequence number for child tasks (shard and range ones) are inherited from parents. There is no need to pass that to the ctor, I will delete it.
And the table would be set to `table_name`.